config.sample.yaml

```diff
@@ -3,6 +3,8 @@
 # run `make auth` to setup the Auth.
 tre_id: __CHANGE_ME__
 location: __CHANGE_ME__
+# Sets the Azure environment to be used. Possible values: public, usgovernment, china, german, stack
+# arm_environment: public
```

Comment:
This needs to be aligned with @LizaShak.

Reply:
Yes, we need it for the core deployment anyway, so from my perspective we will use this var to determine the cloud and propagate it to the terraform